src/main/java/com/example/sharemind/chatMessage/repository/ChatMessageRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.example.sharemind.chatMessage.repository;
+
+import com.example.sharemind.chat.domain.Chat;
+import com.example.sharemind.chatMessage.domain.ChatMessage;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface ChatMessageRepository extends JpaRepository<ChatMessage, Long> {
+    ChatMessage findTopByChatOrderByUpdatedAtDesc(Chat chat);
+
+    int countByChat_ChatIdAndMessageIdGreaterThanAndIsCustomer(Long chatId, Long lastReadMessageId, Boolean isCustomer);
```

Comment:
이것도 findTopByChatOrderByUpdatedAtDesc처럼 Chat으로 카운트하면 될 것 같은데 chatId로 두신 이유가 있나요?
```suggestion
    int countByChatAndMessageIdGreaterThanAndIsCustomer(Chat chat, Long lastReadMessageId, Boolean isCustomer);
```

Reply:
말씀하신 것처럼 Chat으로 넘기는게 훨씬 깔끔하겠네요! 그렇게 수정하였습니다 감사합니다아